Applications/ConsumerApi/test/ConsumerApi.Tests.Integration/StepDefinitions/TokensApiStepDefinitions.cs

```diff
@@ -39,28 +36,26 @@ internal class TokensApiStepDefinitions
         key = "some-value"
     })).BytesRepresentation;
```

Comment:
I know this was already here, but we have an extension method for this conversion I believe:
`private static readonly byte[] CONTENT = JsonConvert.SerializeObject(new { key = "some-value" }).GetBytes();`